controlpanel/api/elasticsearch.py

```diff
@@ -23,3 +24,32 @@ def bucket_hits_aggregation(bucket_name, num_days=None):
                   size=100)
 
     return s.execute().aggregations.bucket_hits
+
+
+def app_logs(app, num_hours=None):
+    if not num_hours:
```

Comment:
maybe I'm missing something, I'm a Python noon after all, but what's the difference between this and having the `num_hours` parameter have a default of `1`: `def app_logs(app, num_hours=1):`?

Reply:
Yep, dunno how I ended up writing this